EventsExpress/ClientApp/src/components/event/event-item-view.js

```diff
@@ -70,16 +71,39 @@ export default class EventItemView extends Component {
         ));
     }
 
-
-    
-
     renderApprovedUsers = (arr, isMyEvent, isMyPrivateEvent) => {
         return arr.map(x => (
             <UserView
```

Comment:
Let's go ahead now and use this new component for `renderOwners`, `renderPendingUsers`, and `renderDeniedUsers` as well.